lib/internal/webstreams/readablestream.js

```diff
@@ -140,6 +140,19 @@ const kError = Symbol('kError');
 const kPull = Symbol('kPull');
 const kRelease = Symbol('kRelease');
 
+let releasedError;
+let releasingError;
+
+function lazyReadableReleasedError() {
+  releasedError ??= new ERR_INVALID_STATE.TypeError('Reader released');
+  return releasedError;
+}
+
+function lazyReadableReleasingError() {
+  releasingError ??= new ERR_INVALID_STATE.TypeError('Releasing reader');
+  return releasingError;
+}
```

Comment:
You'd need to call the `.stack` property, otherwise there is a leak.

Reply:
V8 will keep the whole stack frame in memory and only lazily convert it into a string. If you don't call `stack` it never gets released.

Reply:
https://github.com/nodejs/node/pull/34103#issuecomment-652002364

Reply:
Oh, thank you! Fixed!